src/main/java/gameboy/gamepalyer/GamePlayer.java

```diff
@@ -0,0 +1,12 @@
+package gameboy.gamepalyer;
+
+import gameboy.gamepack.racinggame.RacingGamePack;
+
+public class GamePlayer {
+
+    public static void main(String[] args) {
+        RacingGamePack racingGame = new RacingGamePack();
```

Comment:
게임팩 갈아끼우려면 인터페이스 타입으로 선언하는 게 좋겠네요 🎮
```suggestion
        GamePack racingGame = new RacingGamePack();
```